src/log.py

```diff
@@ -87,20 +87,21 @@ def _update_log_parent_symlinks():
 
     one_year_ago = time.time() - 365 * 24 * 3600
 
-    logs = glob.iglob(OPERATIONS_PATH + "*" + METADATA_FILE_EXT)
+    logs = glob.iglob("*" + METADATA_FILE_EXT, root_dir=OPERATIONS_PATH)
     for log_md in logs:
-        if os.path.getctime(log_md) < one_year_ago:
+        log_file = os.path.join(OPERATIONS_PATH, log_md)
+        if os.path.getctime(log_file) < one_year_ago:
             # Let's ignore files older than one year because hmpf reading a shitload of yml is not free
             continue
 
         name = log_md[: -len(METADATA_FILE_EXT)]
-        parent_symlink = os.path.join(OPERATIONS_PATH, f".{name}.parent.yml")
-        if os.path.islink(parent_symlink):
+        parent_symlink = os.path.join(OPERATIONS_PATH, f"{name}.parent.yml")
+        if not os.path.exists(parent_symlink) or os.path.islink(parent_symlink):
```

Comment:
We must remove this check, because the purpose of this function is to create it^^.

I still have errors during `yunohost log list`:

> 88   ERROR   The YAML metadata file associated with logs is damaged: '20240715-030330-backup_create.parent.yml
Error: File does not exist: '/var/log/yunohost/operations/20240715-030330-backup_create.parent.yml''

Reply:
Something like that
```suggestion
        if (os.path.islink(log_file) and os.path.realpath(log_file) == "/dev/null) or os.path.islink(parent_symlink):
```